### ISSUE TITLE
[Camel 3.11.2, 3.12.0] Consider usage of `IgnoreSplitPackageBuildItem`

### ISSUE THREAD

**Issue by jamesnetherton** [apache/camel-quarkus#2997]
I noticed in the build logs that various places have warnings about split packages, not just for Camel but also for third party dependencies. E.g

```
io.quarkus.arc.deployment.SplitPackageProcessor splitPackageDetection
WARN: Detected a split package usage which is considered a bad practice and should be avoided. Following packages were detected in multiple archives: 
- "org.apache.camel.main" found in [org.apache.camel:camel-base-engine::jar, org.apache.camel:camel-main::jar]
```

Maybe we should consider suppressing some of these?

**Comment by davsclaus** [comment]
The camel-main split package has been fixed for 3.11.2 onwards

**Comment by jamesnetherton** [comment]
I'll close this as the main issue is fixed in Camel.
